### ISSUE TITLE
Pretrained model for testing issue

### ISSUE THREAD

**Issue by xilongzhou** [naoto0804/pytorch-inpainting-with-partial-conv#38]
![result](https://user-images.githubusercontent.com/32085079/71380857-81ba1700-2596-11ea-9af9-077499a63b74.jpg)
Hi
I test the pretrained model you shared online using test.py but the results are very different from yours (as you can see above). There are many artifacts in the masked region. Could you please help me figure this out? Maybe I missed something during the implementation?

Thank you very much!

**Comment by naoto0804** [comment]
Thank you for reporting the issue!
Hmm, I'm not sure... Could you try PyTorch 0.4.1 just in case?

**Comment by xilongzhou** [comment]
Thank you very much for your reply. Yes, My Pytorch vision is 0.4.1 but still get wired results. Did you have any preprocess for the input images? I save images in an image file and directly load them using test.py. Merry Christmas!

**Comment by xilongzhou** [comment]
Hi 
I figured this out. It is because of the interpolation of mask is bi linear instead of nearest neighbor. I tested with mask of size 256*256 but it did not work and then I use 512*512 mask and it works. The pretrained model is trained based on the certain type of mask which is resized from 512*512 using bilinear interpolation, for the 256*256 mask it does not work. I may need to retrain the network using my own mask for testing. 
Thank you very much!
